i21875: Clarify usage of flag for unauthenticated contract IDs (#820)
digital-asset/docs.daml.com: docs/2.10.0/docs/canton/usermanual/upgrading.rst

    canton.participants.<nodeName>.parameters.allow-for-unauthenticated-contract-ids=true

Please adjust ``<nodeName>`` to match your case.

**IMPORTANT**: Please note that you MUST set the flag on ALL the participants connected to the domain.

> Failure to do so will end up in critical failures of the participants.

Reviewer: ```suggestion
Failure to do so causes critical failure of the participants.
```
